components/rating/stories/rating.test.js

```diff
@@ -18,12 +18,24 @@ export const RatingGroup = Variants({
 			isDisabled: true,
 		},
 		{
-			testHeading: "Focused",
-			isFocused: true,
+			testHeading: "Keyboard focused",
+			isKeyboardFocused: true,
 		},
 		{
 			testHeading: "Read-only",
 			isReadOnly: true,
 		},
+		{
+			testHeading: "With tooltip",
+			withTooltip: true,
+		},
+		{
+			testHeading: "Small size",
+			size: "s",
+		},
+		{
+			testHeading: "Medium size",
+			size: "m",
+		},
```

Comment:
We don't need to test sizing separately - this is handled automatically by the testing grid.